k8s/deploy_app.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Now that we're not using this script in the Terraform we should remove it and document the Helm deploy command in the Readme.